src/main/kotlin/no/nav/syfo/personstatus/db/getFromPersonOversiktStatus.kt

```diff
@@ -45,15 +45,16 @@ const val queryHentUbehandledePersonerTilknyttetEnhet = """
                                 dialogmotekandidat = 't' 
                                 AND dialogmotekandidat_generated_at + INTERVAL '7 DAY' < now()
                                 )
-                            OR ( aktivitetskrav = 'NY' OR aktivitetskrav = 'AVVENT')
+                            OR (( aktivitetskrav = 'NY' OR aktivitetskrav = 'AVVENT') AND aktivitetskrav_stoppunkt > ?)
```

Comment:
Jeg er litt usikker på om dette query'et vil klare å bruke den relevante indeksen med denne tilpasningen. I verste fall vil enhetens oversikt bli mye tregere. 

Så siden dette er midlertidig kode tror jeg det vil være bedre å filtrere bort de som ikke skal vises i Kotlin-koden i stedet (loope igjennom søkeresultatet og ta bort de med stoppunkt før cutoff).

Reply:
Vi har hatt en diskusjon på dette, og går for å flytte filtreringen på dato ut av databasen, og inn i appen.